pkg/proxy/winkernel/proxier.go

```diff
@@ -610,6 +692,15 @@ func NewProxier(
 			return nil, fmt.Errorf("source-vip flag not set")
 		}
 
+		if nodeIP.IsUnspecified() {
```

Comment:
Since we are using detectNodeIP that deals with unspecified, is it possible to get an unspecified here?
I couldn't check all code paths, just asking

Reply:
Yes it could be unspecified, in DualStack the bind address doesn't go through detectNodeIp, which only works with IPv4.  This is the same on Linux: https://github.com/kubernetes/kubernetes/blob/90b02692e2980d460fa2e53bf4509df1ec10475f/cmd/kube-proxy/app/server_others.go#L239  https://github.com/kubernetes/kubernetes/blob/90b02692e2980d460fa2e53bf4509df1ec10475f/cmd/kube-proxy/app/server_windows.go#L120